.yarnrc.yml

```diff
@@ -53,30 +53,52 @@ packageExtensions:
       react: '*'
   '@wordpress/annotations@1.24.8':
     peerDependencies:
+      redux: '*'
       react: '*'
   '@wordpress/api-fetch@3.23.1':
     peerDependencies:
       react-native: '*'
   '@wordpress/api-fetch@4.0.0':
     peerDependencies:
       react-native: '*'
+  '@wordpress/api-fetch@5.2.1':
+    peerDependencies:
+      react-native: '*'
```

Comment:
Hmm, do you know why `api-fetch` peer-depends on `react-native`? 🤔

Reply:
Yes, there is one wp package which has `react-native` as a peer dependency, and that causes everything else which uses that package to show a warning. Since a lot of wp packages depend on that package but do not include react-native, yarn says that the peer dependency requirements are not met. Adding it to the peer dependencies in this file basically indicates to yarn that we (the calypso repo) will provide that peer dependency, thus resolving the peer dep requirement.

That said, I think this issue doesn't exist with newer versions of WP packages, so it's possible we can remove it

Reply:
I've just double checked. It hasn't been published yet, but https://github.com/WordPress/gutenberg/pull/34687 will resolve this issue. The exact problem is:

- `react-native-url-polyfill` is added by `@wordpress/url`
- `react-native-url-polyfill` has a peer dep on `react-native`, which isn't provided by `@wordpress/url`
- Everything else including `@wordpress/url` (such as `@wordpress/api-fetch`) now has an invalid peer dependency requirement.
- These are fulfilled by adding `react-native` as a peer dep to each package depending on `@wordpress/url`, and then providing `react-native` at the top level ourselves